playbook/app/pb_kits/playbook/pb_advanced_table/advanced_table.test.jsx

```diff
@@ -1,7 +1,7 @@
 import React, {useState} from "react"
 import { render, screen, waitFor } from "../utilities/test-utils"
 
-import { AdvancedTable } from "../"
+import { AdvancedTable } from "playbook-ui"
```

Comment:
We are now winning here in a big way! Now, anyone can go to a kit example and copy kit code then paste in and use **without** modification. 🎉